_partials/_migrate_awsrds_connect_intermediary.md

```diff
@@ -0,0 +1,92 @@
+
+## Create an intermediary EC2 Ubuntu instance
+<Procedure>
```

Comment:
Do you mean the https://docs-dev.timescale.com/docs-3266-docs-rfc-rewrite-migrate-low-downtime-live-migration-as-migrate-with-near-zero-downtime/migrate/3266-docs-rfc-rewrite-migrate-low-downtime-live-migration-as-migrate-with-near-zero-downtime/live-migration/#create-an-intermediary-ec2-ubuntu-instance and https://docs-dev.timescale.com/docs-3266-docs-rfc-rewrite-migrate-low-downtime-live-migration-as-migrate-with-near-zero-downtime/migrate/3266-docs-rfc-rewrite-migrate-low-downtime-live-migration-as-migrate-with-near-zero-downtime/live-migration/#install-the-psql-client-tools-on-the-intermediary-instance sections. The next AWS sections would not make sense. 

This will make the doc a looot longer. I included the AWS stuff because there is no way to do this without having an AWS machine. Maybe I can add a line in the prerequisites that best practice is to use a AWS VM with psql installed.

Reply:
@billy-the-fish May be we can just include the link? We always recommend our users to use EC2 instance.

Reply:
It was already in the prerequisites, I've bumped the message though: https://docs-dev.timescale.com/docs-3266-docs-rfc-rewrite-migrate-low-downtime-live-migration-as-migrate-with-near-zero-downtime/migrate/3266-docs-rfc-rewrite-migrate-low-downtime-live-migration-as-migrate-with-near-zero-downtime/live-migration/#prerequisites